src/scss/components/_list.scss

```diff
@@ -100,4 +109,97 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: get-weight(bold);
+    text-align: left;
+    text-transform: uppercase;
+  }
+
+  &--on-column {
+    &[class*='col-'] {
+      display: inline-block;
```

Comment:
The right approach is to change the direction of flex items, not change the display of them.

Reply:
Right, but the parent of this columns must be a flex :wink